resources/public/api/conf/1.0/application.yaml

```diff
@@ -905,8 +905,20 @@ paths:
       security:
       - userRestricted:
         - read:pillar2
-components:
-  schemas:
+      security:
+      - userRestricted:
+        - write:pillar2
+          type: string
+          format: date
+
+      required:
+      - accountingPeriodFrom
+      - accountingPeriodTo
+      - filedDateGIR
+      - countryGIR
+      - reportingEntityName
+      - TIN
```

Comment:
Is TIN used elsewhere? Can we use a more meaningful name or is it a term/acronym widely used/known?